app/cmd/plugin_search.go

```diff
@@ -93,3 +193,14 @@ func (o *PluginSearchOption) Output(obj interface{}) (data []byte, err error) {
 	}
 	return
 }
+
+func formatTab(table *util.Table, i int, plugin client.CenterPlugin) {
```

Comment:
`formatTab` can rename to `formatTable`. This is will be more clear.

Reply:
Okay. So we maybe make a development specification. For example, how to name it?